src/configure

```diff
@@ -27,7 +27,7 @@ CONFIGURE_VERSION=2
 
 function rel2abs {
   if [ ! -z "$1" ]; then
-    local retval=`cd $1 2>/dev/null && pwd || exit 1`
```

Comment:
Ok, I made the change because it failed for directories which don't yet exist, which I needed to specify an install prefix using --prefix.